x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -384,6 +384,44 @@ var testCases = []struct {
 			`{"foo":"a","page":"0"}`, `{"foo":"b","page":"1"}`, `{"foo":"c","page":"0"}`, `{"foo":"d","page":"0"}`,
 		},
 	},
+	{
+		name: "pagination_not_log_fail",
+		setupServer: func(t testing.TB, h http.HandlerFunc, config map[string]interface{}) {
+			server := httptest.NewServer(h)
+			config["request.url"] = server.URL
+			t.Cleanup(server.Close)
+		},
+		baseConfig: map[string]interface{}{
+			"interval":       time.Millisecond,
+			"request.method": http.MethodGet,
+			"response.split": map[string]interface{}{
+				"target": "body.items",
+				"transforms": []interface{}{
+					map[string]interface{}{
+						"set": map[string]interface{}{
+							"target": "body.page",
+							"value":  "[[.last_response.page]]",
+						},
+					},
+				},
```

Comment:
This split transform don't seem relevant to the test.

Reply:
The test here is a direct comparison with the test above. I wanted to make it as close as possible to that with just the change in this behaviour so that they could be compared. The same is true of the other test addition here.